### PR TITLE
creating sections to structure the page

### PR DESCRIPTION
**Pull request by johangiraud**
dunno how to do that without having these containers. I am open to suggestions in case you have some

### CODE REVIEW

File: templates/profile.html [hunk 1]
```diff
@@ -7,98 +7,105 @@
 <h1>Create a profile</h1>
 {% endif %}
 
-<ul>
-  <li>Full name: <span data-edit-name=name data-edit-attrs="required">{{ user.profile.name or '' }}</span></li>
-  <li>Website: <span data-edit-name=website data-edit-attrs="type=url"><a href="{{ user.profile.website }}" target=_blank>{{ user.profile.website }}</a></span></li>
-</ul>
+<main>
```
Comment (vinyll) [inline templates/profile.html]: I suppose that main should be in the base layout and eventually the `<h1>` title inside of it.

File: templates/profile.html [hunk 2]
```diff
@@ -7,98 +7,105 @@
...
+        .catch(e => alert(e.error))
       })
-      .catch(e => alert(e.error))
-    })
-  </script>
-{% endif %}
+    </script>
+  {% endif %}
+  </section>
+  <section>
```
Comment (vinyll) [inline templates/profile.html]: Should this  be `<aside>`?

File: templates/profile.html [hunk 3]
```diff
@@ -7,98 +7,105 @@
...
+    <h2>Contributions</h2>
+  </section>
+</main>
```
Comment (vinyll) [inline templates/profile.html]: We want the `<script>` at the end of the document.

**Review by vinyll — CHANGES_REQUESTED** [review]

File: templates/profile.html (hunk 1 above)
Comment (johangiraud) [reply templates/profile.html]: sure

File: templates/profile.html (hunk 2 above)
Comment (johangiraud) [reply templates/profile.html]: to me `aside` means that is a major column and a minor one : `aside`. Here it does't make sense cause both columns have the same importance. What do you think ?

File: templates/profile.html (hunk 3 above)
Comment (johangiraud) [reply templates/profile.html]: ok, why don't you place this `<script>` in an external file like script.js or utils.js ?

Comment (vinyll) [reply templates/profile.html]: This page is unique and the script is specific.
We could therefore place the script in a `profile.js` file if you'd rather have it outside.

File: templates/base.html [hunk 4]
```diff
@@ -38,5 +38,7 @@
 
 {% include "security/_messages.html" %}
 {% block body %}{% endblock %}
-
+    
+<main>
```
Comment (vinyll) [inline templates/base.html]: Taking the opening tag also mean you should take the closing tag along.

**Review by vinyll — CHANGES_REQUESTED** [review]

File: templates/profile.html (hunk 3 above)
Comment (johangiraud) [reply templates/profile.html]: yo no prob if the script is used only in this page, it makes sense then

**Review by vinyll — APPROVED** [review]

File: templates/base.html (hunk 4 above)
Comment (johangiraud) [reply templates/base.html]: 👍

**Pull request merged**